### PR TITLE
Feat/add vite base

### LINKED ISSUE #34: [feature]: Add "base public path" configuration
If i configure my vite frontend with base like:
```ts
export default defineConfig({
  //...
  base: '/app'
  //...
});
```
I would like to receive from inertia the url's associated in the template: `http://localhost:5173/app/src/index.tsx` and `http://localhost:5174/app/@vite/client`

Proposal:
```rust
        let inertia = vite::Development::default()
            .port(5173)
            .main("src/index.tsx")
            .lang("en")
            .title("My inertia app")
            .https(true)
            .base("/app") // add base
            .into_config();
```

I can open a PR for it.

### PR DESCRIPTION
**Pull request by Dsaquel**
fixes #34

### CODE REVIEW

File: src/vite.rs [hunk 1]
```diff
@@ -46,6 +47,7 @@ pub struct Development {
 impl Default for Development {
     fn default() -> Self {
         Development {
+            base: "/",
```
Comment (mjhoy) [inline src/vite.rs:50]: Can we make this an empty string by default, and require that users include the preceding slash but not the following (e.g., `.base("/app")`? I'm looking at `vite-ruby` as an example here: https://vite-ruby.netlify.app/config/#base

File: src/vite.rs [hunk 2]
```diff
@@ -57,6 +59,18 @@ impl Default for Development {
 }
 
 impl Development {
+    /// ```rust
```
Comment (mjhoy) [inline src/vite.rs:62]: Awesome, thanks for adding documentation!

Maybe add a one line description as well? Something like:

> Adds a path as a prefix to included vite client assets, e.g., `"/app"`.

Or if you have a better way of phrasing that :)

**Review by mjhoy — CHANGES_REQUESTED** [review]
This is great, thanks! I just had one note. Can you also add a note to the changelog under the unreleased section?

File: src/vite.rs (hunk 1 above)
Comment (Dsaquel) [reply src/vite.rs:50]: Make more sense 👍

**Review by mjhoy — APPROVED** [review]
awesome, lgtm

**Pull request merged**